tests/puppeteer/campaigns/sanity/03_ordersBO/02_editOrder.js

```diff
@@ -41,15 +43,17 @@ describe('Edit Order BO', async () => {
   // Steps
   loginCommon.loginBO();
   it('should go to the Orders page', async function () {
-    await this.pageObjects.boBasePage.goToSubMenu(this.pageObjects.boBasePage.ordersParentLink,
-      this.pageObjects.orderPage.ordersLink);
-    const pageTitle = await this.pageObjects.orderPage.getPageTitle();
-    await expect(pageTitle).to.contains(this.pageObjects.orderPage.pageTitle);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.ordersParentLink,
+      this.pageObjects.boBasePage.ordersLink,
+    );
+    const pageTitle = await this.pageObjects.ordersPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.ordersPage.pageTitle);
   });
   it('should go to the first order page', async function () {
-    await this.pageObjects.boBasePage.waitForSelectorAndClick(this.pageObjects.orderPage.orderfirstLineIdTD);
+    await this.pageObjects.ordersPage.goToOrder('1');
```

Comment:
What do you mean exactly by 'in configuration' ?

Reply:
Exactly the same as you do for html selector, but for objects values.
If you use this id many times, maybe you should create
DEFAULT_ORDER_ID = 1
Because if one day you add scenario with many others id, it will be easier for everyone to understand why the default order id is X, and not Y.
Same for products, if you want a customizable product, it's maybe not 1 (I really don't know), and if you want a product with combination, you will take the 8 (Still don't know :sweat_smile: ), adding "words" to defined which object you'll use will help everybody to understand what you're doing, just by reading.

Reply:
my bad, missreading :/